R/manual_layer_configure.R

```diff
@@ -14,11 +14,9 @@
     cfg <- jsonlite::fromJSON(cfgfile)
 }
 
-.storeConfig <- function() {
-    homedir <- Sys.getenv("HOME")
+.storeConfig <- function(homedir=Sys.getenv("HOME")) {
     if (homedir == "") {
-        warning("No HOME environment variable.")  # Windows ?
-        return(NULL)
+        stop("No HOME environment variable or homedir value.", call .= FALSE)  # Windows ?
```

Comment:
I may have had a typo in my comment, you need `call. = FALSE` here (and I now force myself to make a space as I otherwise get a ligature for `.=` when there shouldn't be one -- `call.` is just an oddly named variable)